editor/question.js

```diff
@@ -502,4 +502,26 @@ angular.module('org.ekstep.question', ['org.ekstep.metadataform'])
     ctrl.init();
   }]);
 
+  angular.module('org.ekstep.question').service('questionServices', ['$http', function ($http) {
```

Comment:
Why do you need an angular service for the below methods. It can be just controller methods.

Reply:
The below two functionality 

**invokeAssetBrowser**
`ecEditor.dispatchEvent('org.ekstep.assetbrowser:show`

**generateTelemetry**
`ecEditor.getService('telemetry').interact(` 

were used inside **mtf** and **ftb** editor plugins, 
So rather having the functionality inside two different angular applications 

`angular.module('ftbApp')`
`angular.module('mtfApp')` 

We wrote a service in
`angular.module('org.ekstep.question')` 

So that `ftbApp` and `mtfApp` angular apps can access the same service.

Also please mention if there is any better way